src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.scss

```diff
@@ -237,3 +237,47 @@
     background-color: var(--after-primary-light-grey);
   }
 }
+
+@media (max-width: 480px) {
+  .content {
+    margin: 0;
+    padding-bottom: 100px;
+  }
+
+  .header-list {
+    display: flex;
+    justify-content: space-between;
+    margin-top: -32px;
+
+    div {
+      color: var(--ubs-primary-grey);
+      font-size: 14px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 20px;
+      letter-spacing: 0.10000000149011612px;
```

Comment:
What a strange number
You should change it to something normal

Reply:
Done